### PR TITLE
Add net461 and net462 targets

### LINKED ISSUE #9: Add net461 target
As it says on the tin.

Add a build target for net461, optionally other netfx versions as well. Other netfx versions after 461 should work with netstandard, but support for netstandard on netfx is buggy to put it lightly.

### PR DESCRIPTION
**Pull request by TheBrambleShark**
Fixes #9

Blocked by Remora/Remora.SDK#3 (build will not succeed until this issue is fixed)

### CODE REVIEW

**Comment by Nihlus** [comment]
Is net462 required? The library already builds for netstandard2.0, which claims support for .NET 4.6.2.

**Comment by Nihlus** [comment]
Also, TupleElementNamesAttribute is still missing with PolySharp installed. I've switched to it in Remora.Sdk anyway because it's more feature-complete, but it seems like this is one piece that's still missing.

**Comment by Nihlus** [comment]
Follow-up issue: installing System.ValueTuple conditionally fixes that issue, but now I'm getting multiple definitions of IsExternalInit instead.

Related: https://github.com/Sergio0694/PolySharp/issues/48

**Comment by TheBrambleShark** [comment]
> Is net462 required? The library already builds for netstandard2.0, which claims support for .NET 4.6.2.

I was looking into this again - Microsoft attempted to retroactively add support for net461 to netstandard2.0, which completely broke and they [never actually went and fixed it](https://stu.dev/dotnet-framework-support-for-dotnetnet-standard-2-0/). Net462 was not affected and will receive support through netstandard2.0.

TL;DR: No, it is not required. Net461 is required because of bugs with the interface between Net461 and Netstandard2.0.

**Comment by Nihlus** [comment]
I've pushed some experimental changes that don't require PolySharp and it seems to work okay on my end. Can you test and make sure?

**Comment by TheBrambleShark** [comment]
I created a test project in UiPath C# (the more finnicky of the two on this version) with effectively this code:
```
Result<string> result = "Success!";
MessageBox.Show(result.Entity);
```
And the message box appeared as expected.

**Comment by Nihlus** [comment]
I'll close this, then.
